internal/operands/common-templates/reconcile.go

```diff
@@ -0,0 +1,89 @@
+package common_templates
+
+import (
+	templatev1 "github.com/openshift/api/template/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"kubevirt.io/ssp-operator/internal/common"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func AddWatchTypesToScheme(s *runtime.Scheme) error {
+	return templatev1.Install(s)
+}
+
+func WatchClusterTypes() []runtime.Object {
+	return []runtime.Object{
+		&rbac.ClusterRole{},
+		&rbac.ClusterRoleBinding{},
+		&templatev1.Template{},
```

Comment:
The old operator installs templates to a namespace. The file `common-templates-v0.12.1.yaml` does not contain namespaces, because they depend on SSP resoource. This operator should also create templates in a namespace.

Reply:
Changed now

Reply:
Currently, when HCO deploys the common-templates CR, it does so in the `openshift` namespace, thus templates are deployed in the `openshift` namespace as well. I think our SSP CR should have something like this:
```
spec:
  commonTemplates:
    namepace: $NAMESPACE
```
So we can control in which namespace the templates are deployed.